src/Makefile

```diff
@@ -21,6 +21,7 @@ FLAGS := \
 	-D_GNU_SOURCE -I$(BUILDDIR) -I$(SRCDIR) \
 	-I$(SRCDIR)/flisp -I$(SRCDIR)/support \
 	-I$(LIBUV_INC) -I$(build_includedir) -DLIBRARY_EXPORTS \
+	-I$(LOCALBASE)/include \
```

Comment:
On my system it is:

```
`gcc -print-prog-name=cc1plus` -v
ignoring nonexistent directory "/usr/lib/gcc/x86_64-pc-linux-gnu/6.3.1/../../../../x86_64-pc-linux-gnu/include"
#include "..." search starts here:
#include <...> search starts here:
 /usr/lib/gcc/x86_64-pc-linux-gnu/6.3.1/../../../../include/c++/6.3.1
 /usr/lib/gcc/x86_64-pc-linux-gnu/6.3.1/../../../../include/c++/6.3.1/x86_64-pc-linux-gnu
 /usr/lib/gcc/x86_64-pc-linux-gnu/6.3.1/../../../../include/c++/6.3.1/backward
 /usr/lib/gcc/x86_64-pc-linux-gnu/6.3.1/include
 /usr/local/include
 /usr/lib/gcc/x86_64-pc-linux-gnu/6.3.1/include-fixed
 /usr/include
End of search list.
```

So both `/usr/include` and `/usr/local/include` are on the default list, but I don't think there is anything requiring it to be, besides convention.